app/models/listing.rb

```diff
@@ -1,12 +1,17 @@
 class Listing < ActiveRecord::Base
   belongs_to :user
 
-  def self.allHomeType
-    [["House", 0], ["Condo", 1], ["Townhouse", 2]]
-  end
+  # def self.allHomeType
+  #   [["Condo", 0], ["House", 1], ["Townhouse", 2]]
+  # end
+
+  # def self.allNeighbourhoods
+  #   [["Gastown", 0], ["Yaletown", 1], ["Chinatown", 2], ["Kitsilano", 3], ["Main Street", 4]]
+  # end
+
+  # def self.areas
+
+  # end
```

Comment:
Remove un-used code, you don't need to keep is as a comment you can find it in your git history